projects/storefrontlib/shared/components/chat-messaging/messaging/messaging.model.ts

```diff
@@ -16,6 +16,7 @@ export interface MessagingConfigs {
   charactersLimit?: number;
   newMessagePlaceHolder?: string;
   enableFileUploadOption?: boolean;
+  dateFormat?: string;
```

Comment:
Can the hosting component set the format?
It will be preferable if that is the case so that customer ticketing can set a general date format that will affect all its pages.

Reply:
We did all the changes required for dateFormat in this [PR](https://github.com/SAP/spartacus/pull/16113/commits/b2a443e3ce8ac05feef5cb8793c25d679deaa197)